App.tsx

```diff
@@ -1,12 +1,48 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
 
 export default function App() {
-  return (
-    <View style={styles.container}>
-      <Text>Hello World!!!</Text>
-    </View>
-  );
+  const [vulpix, setVulpix] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  const getVulpix = async () => {
+    try {
+      const res = await fetch('https://pokeapi.co/api/v2/pokemon/37');
+      const data = await res.json();
+
+      setVulpix(data);
+      setLoading(false);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getVulpix();
```

Comment:
nit -> this is more about learning how exactly hooks work in React rendering, but in this case it would be a bit better to either write custom hook (e.g. `useVulpixLoader`) or move `getVulpix` implementation inside `useEffect`.

Otherwise `getVulpix` function is created in every `render` (when you write arrow function you always create a new function), but it's used only once (in `useEffect` a.k.a. `componentDidMount`).

However, this component will be rarely rerendered, so it's fine to keep it as it is for now and improve in the future 😄